client/main.js

```diff
@@ -27,9 +27,9 @@ __webpack_public_path__ = global.wcsPluginData.assetPath;
 
 // We need to lazy load the moment locale files.
 // First we try language code with region if it's different then fall back to language code only.
-if ( window.i18nLocale && window.i18nLocale.localeSlug !== 'en' ) {
+if ( window.i18nLocale && window.i18nLocale.localeSlug !== 'en-US' && window.i18nLocale.localeSlug !== 'en' ) {
```

Comment:
What about something like:

```if ( window.i18nLocale && ! [ 'en-US', 'en' ].includes( window.i18nLocale.localeSlug )``` 

?

IMHO is less verbose and easier to read.

Reply:
I like it!

Reply:
Updated!